File: Récupération 20210215/Facility Management 1.4.1.7/Form/GetElements.vb
Public Class GetElements


    Private Sub ComboBoxVersion_KeyPress(sender As Object, e As System.Windows.Forms.KeyPressEventArgs) Handles ComboBoxVersion.KeyPress
        e.KeyChar = Chr(0)
    End Sub

    Private Sub ButtonGet_Click(sender As System.Object, e As System.EventArgs) Handles ButtonGet.Click
        Me.Close()
    End Sub

    Private Sub ButtonCancel_Click(sender As System.Object, e As System.EventArgs) Handles ButtonCancel.Click
        Me.Close()
    End Sub

    Private Sub ButtonIgnore_Click(sender As System.Object, e As System.EventArgs) Handles ButtonIgnore.Click
        Me.Close()
    End Sub
End Class

File: Facility Management_Decompiler/Facility Management/GetElements.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Facility_Management
{
    public partial class GetElements : Form
    {

		public GetElements()
		{
			InitializeComponent();
		}
		
		private void ComboBoxVersion_KeyPress(object sender, KeyPressEventArgs e)
		{
			e.KeyChar = '\0';
		}

		private void ButtonGet_Click(object sender, EventArgs e)
		{
			Close();
		}

		private void ButtonCancel_Click(object sender, EventArgs e)
		{
			Close();
		}

		private void ButtonIgnore_Click(object sender, EventArgs e)
		{
			Close();
		}
	}
}
